gridstatus/ercot.py

```diff
@@ -1851,10 +1851,20 @@ def _download_html_table(self, url, verbose=False):
             1
         ]  # Split the string on ': ' to get just the time part
 
+        now = pd.Timestamp.now(tz=self.default_timezone)
+
+        # Determine if during the repeated DST hour. Pandas wants ambiguous=True if
+        # it is during the repeated hour. US/Central is UTC-6 during standard time
+        # and UTC-5 during DST.
+        ambiguous = (now.utcoffset().total_seconds() / 3600) == -5.0
```

Comment:
I think this works because Pandas only cares about ambiguous during the repeated hour.

Reply:
"Pandas wants ambiguous=True if the time is DST during the repeated hour. US/Central is UTC-6 during standard time"

doesn't this imply we want 

```
ambiguous = (now.utcoffset().total_seconds() / 3600) == -6.0
```

Reply:
I don't think so because 6 hours is the offset outside of DST. We want ambiguous to be True when the timezone is in DST (offset 5 hours). 

From the Pandas docs:

> bool-ndarray where True signifies a DST time, False designates a non-DST time (note that this flag is only applicable for ambiguous times)

Example for US/Central

* 0130-0500 in DST, ambiguous=True
* 0130-0600 not in DST, ambiguous=False

Reply:
i think you have those switched

![image](https://github.com/user-attachments/assets/11c3a38f-d3ba-455e-8392-2b5f46f4a922)

Reply:
oh wait im mixing things up. CST is "standard time" not "savings time" 🤦

Reply:
i think your comment is wrong/ambiguous you say: "Pandas wants ambiguous=True if it is during the repeated hour"

the non-DST time is when the hour repeats itself, so i would think a better might just be to copy pandas documentation: "True signifies a DST time, False designates a non-DST time"

Reply:
The comment currently reads:

> Pandas wants ambiguous=True if the time is DST during the repeated hour.

Reply:
looks good. sorry for my confusion here.

Reply:
No worries, ERCOT makes this really confusing.